dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1707,6 +1707,15 @@ public void deleteAllVersionsandBackup(List<Contentlet> contentlets, User user,
         }
         deleteBinaryFiles(contentletsVersion,null);
 
+        try {
+            String bla = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "bla" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)